### PR TITLE
Add Digital Asset Management.sol

### PR DESCRIPTION
**Pull request by TGBinh**

### CODE REVIEW

File: Digital Asset Management.sol [hunk 1]
```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.3;
+
+interface iDigitalAsset {
```
Comment (conalpham) [inline Digital Asset Management.sol:4]: Name of Interface and contract should be started by uppercase

File: Digital Asset Management.sol [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        } 
+        else {
+            revert("Invalid asset type");
+        }
+
+        uint256 id = assetCount; 
+        assetCount++;
+        assets[id] = newAsset;
+    }
+
```
Comment (conalpham) [inline Digital Asset Management.sol]: Add function to get data from assets list

File: Digital Asset Management.sol (hunk 1 above)
Comment (TGBinh) [reply Digital Asset Management.sol:4]: Oh, new knowledge

File: Digital Asset Management.sol (hunk 2 above)
Comment (TGBinh) [reply Digital Asset Management.sol]: I have added it already

**Pull request merged**